tasks/install-Debian.yml

```diff
@@ -19,3 +19,8 @@
   package:
     name: rake
     state: present
+
+- name: Install phantomjs
```

Comment:
probably, it is better to make them optional, like

```yaml
- name: Install jenkins_extra_packages
  package: "{{ item }}"
    state: present
  with_items: "{{ jenkins_extra_packages }}"
``` 

and a similar task for npm packages.